transformation_test/transformation_test.go

```diff
@@ -155,7 +152,8 @@ func checkOutput(t *testing.T, name string, got []map[string]any) {
 	if err := yaml.Unmarshal(wantBytes, &want); err != nil {
 		t.Fatal(err)
 	}
-	if diff := cmp.Diff(got, want); diff != "" {
+
+	if diff := cmp.Diff(gotBytes, wantBytes); diff != "" {
```

Comment:
Why are you diffing gotBytes and wantBytes instead of got and want? I believe that will make the diff much less readable.